pkg/api/l2geth.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/kubernetes"
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
+)
+
+type l2gethConfig struct {
+	data map[string]string
+}
+
+func (t *TitondAPI) CreateL2Geth(data *model.Component /*TODO : get params for config, namespace*/) (*model.Component, error) {
+	// TODO : deal with DB
+	// t.db.CreateComponent()
+
+	/*
+		This is currently hardcoding, but
+		will be replaced by taking a value as a parameter and generating it
+	*/
+	namespace := "default"
+	config := &l2gethConfig{
+		data: map[string]string{},
+	}
+
+	config.data["ETH1_CONFIRMATION_DEPTH"] = "1"
+	config.data["GASPRICE"] = "100"
+
+	l2GethChan := make(chan error)
+
+	go t.createL2Geth(namespace, config, l2GethChan)
+
+	if err := <-l2GethChan; err != nil {
```

Comment:
Does it block the execution? We don't have to return err for this goroutine. I think it's enough to print error only in the goroutine.

Reply:
i fixed it! thanks!